api/descriptions/custom-rules/custom-rules.md

```diff
@@ -1,2 +1,2 @@
-Custom runtime rules define precise runtime defense actions for containers, hosts, and Kubernetes audits.
+Defines a precise defense action for containers, hosts, and app-embedded audits.
```

Comment:
@ssugandh Revert back to "Defines precise runtime defense actions for containers, hosts, and Kubernetes audits."

"Kubernetes audits" is a product feature.

Reply:
@iansk : I reverted. I corrected due to the change in the UI.
![image](https://user-images.githubusercontent.com/91111475/155334554-8813df11-ea36-4d01-bb5e-3e5c92bad3cf.png)

Reply:
@ssugandh Good point. So we actually need to extend the description to include WAAS requests and WAAS responses. Also, we spell "Kubernetes" with a capital "K". Please update.

Reply:
@ssugandh What about WAAS custom rules? This endpoint is also used for that